tests/scripts/gen_release_notes.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+set -e
+
+function help() {
+  print="
+  EXAMPLE command:
+  sh tests/scripts/gen_release_notes.sh upstream/release-1.6 v1.6.7
+  "
+  echo "$print"
+}
+
+trap help Exit
+
+FROM_BRANCH=$1 # example branch name, "upstream/release-1.6"
+TO_BRANCH=$2 # example branch name, "v1.6.7"
+
+if [[ $# -ne 2 ]]; then
+  echo "requires exactly 2 argument is required"
+  exit
+elif [ -z "${GITHUB_USER}" ] || [ -z "${GITHUB_TOKEN}" ]; then
+  echo "requires both GITHUB_USER and GITHUB_TOKEN to be set as env variable"
+  exit
+fi
+
+
+# for releases notes
+function release_notes() {
+  for pr in $(git log --pretty="%s" --merges --left-only ${FROM_BRANCH}...${TO_BRANCH} | grep pull | awk '{print $4}' | cut -c 2-); do
+  #echo "PR: $pr"
+  backport_pr=$(curl -s -u ${GITHUB_USER}:${GITHUB_TOKEN} https://api.github.com/repos/rook/rook/pulls/${pr} | jq '.title')
+  # check if it is manual backport or not
+  if [[ "$backport_pr" =~ .*"(backport".* ]]; then
+    #echo "PR $pr --> $backport_pr"
+    #find the PR number after the #
+    original_pr=$(echo $backport_pr | sed -n -e 's/^.*#//p' | grep -E0o '[0-9]' | tr -d '\n')
+  else
+    original_pr=$pr
+  fi
+  #echo "Looking up PR $original_pr"
```

Comment:
Is it needed?

Reply:
it is needed when we do manual backports.

Reply:
Ok but do you need to uncomment it?

Reply:
I have put the better comment now, earlier it was a comment for testing.